config/initializers/carrierwave.rb

```diff
@@ -34,7 +34,7 @@ def strip
 # Additional carrierwave configurations
 CarrierWave.configure do |config|
   config.cache_dir = Rails.root.join("tmp", "uploads")
-  config.storage :file
+  config.storage = :file
```

Comment:
Ha, what? And we don't need to put `storage :fog` in production?

Reply:
The prod value was being set twice:
https://github.com/bikeindex/bike_index/blob/cde3178651a00100803ebd97ed857f2e3e3fc097/config/initializers/carrierwave.rb#L35-L43